src/apps/bm_devkit/serial_bridge/CMakeLists.txt

```diff
@@ -65,7 +65,6 @@ set(LIB_FILES
     ${SRC_DIR}/lib/debug/debug_bm_service.cpp
     ${SRC_DIR}/lib/debug/debug_configuration.cpp
     ${SRC_DIR}/lib/drivers/abstract/abstract_i2c.cpp
-    ${SRC_DIR}/lib/drivers/adin2111/src/adi_bsp.c
```

Comment:
should the other adi_mac/phy/spi_oa files be removed like in the other apps cmakelists?

Should the BSP_IONCLUDES be included as well?

Same comment applies to the rx_live_example.

Reply:
I believe @towynlin is already working on the rx live example 👍

Reply:
Yep, and now fixed this one too. Thanks!